pkg/migration/controllers/migration.go

```diff
@@ -1688,6 +1702,15 @@ func (m *MigrationController) updateOwnerReferenceOnPVC(
 			var dynamicClient dynamic.ResourceInterface
 			ownerName := ""
 			for _, o := range objects {
+				gkv := o.GetObjectKind().GroupVersionKind()
+				keyName := fmt.Sprintf("%v-%v-%v", gkv.Group, gkv.Kind, gkv.Version)
```

Comment:
nit: may be write a helper function to get the key name for stashCRMap and use it in both the places.

Reply:
Done